frontend/src/tabs/authorship.pug

```diff
@@ -14,6 +14,7 @@
     .period.medium-font
       span &#8627; &nbsp;
       span {{ info.minDate }} to {{ info.maxDate }}
+  .title
```

Comment:
what's the reason for adding this?

Reply:
This is present in the original CSS. So I have kept it as it is, and just removed `group-name`, `author` and `period` from it.